### DISCUSSION TITLE
How should I update the Dockerfile when I use nextjs 13? The

### DISCUSSION THREAD

**Discussion by TinsFox** [vercel/next.js#42820] [Help]
How should I update the Dockerfile when I use nextjs 13? The product after upgrading to nextjs 13 no longer has a **standalone** directory. 
Here is nextjs 12
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/33956589/201454961-9ddfcc1d-e2f8-4f18-ba72-d902dc2a19b7.png">
Here is nextjs 13
<img width="494" alt="image" src="https://user-images.githubusercontent.com/33956589/201454984-56effa54-7036-4716-b4fa-f30be46d3b13.png">

**Comment by kylemclaren** [comment]
Also struggling with this - there is some indication that standalone is not yet supported.

Try comment it out and add `COPY --from=builder --chown=nextjs:nodejs /app/.next/server ./.next/server`
